diff --git a/node_modules/html2pdf.js/dist/html2pdf.js b/node_modules/html2pdf.js/dist/html2pdf.js
index b01e7e6..3b7ebdf 100644
--- a/node_modules/html2pdf.js/dist/html2pdf.js
+++ b/node_modules/html2pdf.js/dist/html2pdf.js
@@ -345,6 +345,7 @@ _worker_js__WEBPACK_IMPORTED_MODULE_5__.default.prototype.toContainer = function
     var pxPageHeight = this.prop.pageSize.inner.px.height; // Check all requested modes.
 
     var modeSrc = [].concat(this.opt.pagebreak.mode);
+    var marginTop = this.opt.margin[0];
     var mode = {
       avoidAll: modeSrc.indexOf('avoid-all') !== -1,
       css: modeSrc.indexOf('css') !== -1,
@@ -411,7 +412,7 @@ _worker_js__WEBPACK_IMPORTED_MODULE_5__.default.prototype.toContainer = function
         var pad = (0,_utils_js__WEBPACK_IMPORTED_MODULE_6__.createElement)('div', {
           style: {
             display: 'block',
-            height: pxPageHeight - clientRect.top % pxPageHeight + 'px'
+            height: marginTop + pxPageHeight - clientRect.top % pxPageHeight + 'px'
           }
         });
         el.parentNode.insertBefore(pad, el);
